build.gradle

```diff
@@ -1,18 +1,23 @@
 plugins {
     id 'java'
     id 'scala'
+    id 'application'
 }
 
 group 'com.example'
 version '1.0'
 
 sourceCompatibility = 1.8
 
+application {
+    mainClassName = project.hasProperty("mainClass") ? project.getProperty("mainClass") : "NULL"
+}
+
 repositories {
     mavenCentral()
     mavenLocal()
     maven {
-        url 'http://files.couchbase.com/maven2'
+        url 'https://files.couchbase.com/maven2'
```

Comment:
Got an error when running gradle: 
```
Using insecure protocols with repositories, without explicit opt-in, is unsupported.
```

So I switched this to https and it was happy.